lldb/include/lldb/Core/DemangledNameInfo.h

```diff
@@ -159,4 +160,23 @@ struct TrackingOutputBuffer : public llvm::itanium_demangle::OutputBuffer {
 };
 } // namespace lldb_private
 
+/// Custom deleter to use with unique_ptr.
+///
+/// Usage:
+/// \code{.cpp}
+///
+/// auto OB =
+///     std::unique_ptr<TrackingOutputBuffer, TrackingOutputBufferDeleter>(
+///         new TrackingOutputBuffer());
+///
+/// \endcode
+struct TrackingOutputBufferDeleter {
```

Comment:
Can we just put it into the test file for now? And put it back into the header once we actually need it elsewhere?